recipes/r-constructive/meta.yaml

```
@@ -0,0 +1,85 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-constructive
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/constructive_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/constructive/constructive_{{ version }}.tar.gz
+  sha256: 307a540d91ec486c08f1e1dc82a228c54a6350d07ea635f3252a917a61f3e5f0
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```

Missing DSO's:

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'        # [win]
    - '*/Rblas.dll'    # [win]
    - '*/Rlapack.dll'  # [win]

```

Note that the latest version of `conda_r_skeleton_helper` includes this by default - be sure to update your copy!